.travis.yml

```diff
@@ -1,9 +1,48 @@
 language: c
 dist: trusty
+script: bash -ex travis.sh
```

Comment:
This is a really minor thing, but I think for me it would make the file easier to follow if these properties followed the execution order (`before_install` first, then `script` after that)